intelmq/bots/experts/idea/expert.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+from collections import Sequence, Mapping
+from base64 import b64decode
+from uuid import uuid4
+from urllib.parse import quote_plus
+
+from intelmq.lib.bot import Bot
+
+
+def quot(s):
+    return quote_plus(s or "")
+
+
+def addr4(s):
```

Comment:
please use the python 3 ip address type functions.
https://docs.python.org/3/howto/ipaddress.html#ipaddress-howto